--- a/u-boot/Makefile
+++ b/u-boot/Makefile
@@ -501,6 +501,10 @@ domino_config : unconfig hornet_common_c
 	# uses 20 MHz oscillator
 	#@echo "#define CONFIG_40MHZ_XTAL_SUPPORT            1" >> include/config.h
 
+	@echo "#define GPIO_6_ENABLE_USB                    6" >> include/config.h
+	@echo "#define GPIO_6_ON                            1" >> include/config.h
+	@echo "#define GPIO_13_INDICATE_POWER              13" >> include/config.h
+	@echo "#define GPIO_13_ON                           1" >> include/config.h
 	@echo "#define GPIO_WLAN_LED_BIT                    0" >> include/config.h
 	@echo "#define GPIO_WLAN_LED_ON                     1" >> include/config.h
 	@echo "#define GPIO_RST_BUTTON_BIT                 11" >> include/config.h
